tests/test_distortion.py

```diff
@@ -70,11 +77,11 @@ def test_ms_ssim_module(num_channels, img_size, batch_sizes):
     metric = MultiscaleStructuralSimilarity()
 
     imgs1 = [
-        torch.rand([batch_size, num_channels, img_size, img_size])
+        rand_im([batch_size, num_channels, img_size, img_size], 123)
```

Comment:
was the intention here to use the same seed for every tensor in the list? I don't think this matters based on the specific inputs sizes we have, but if `batch_sizes` has duplicate entries then duplicate tensors will show up here.

Reply:
Nice catch - this is a bug.